resources/views/courses/create.blade.php

```diff
@@ -18,26 +18,26 @@
 			<label for="teachers">Teacher(s)</label>
 			<select class="ui fluid search dropdown" name="teachers[]" id="teachers" multiple>
 				@foreach ($teachers as $teacher)
-					<option value={{ $teacher['id'] }} {{ ($teacher['id'] == Auth::user()->id) ? 'selected' : '' }}> {{ $teacher['name'] }} </option>
+					<option value={{ $teacher->id }} {{ ($teacher->id == Auth::id()) ? "selected" : '' }}> {{ $teacher->name }} </option>
```

Comment:
```suggestion
<option value="{{ $teacher->id }}" ...></option>
```

Reply:
It's the **`value`** that needs the double quoting, not the `selected`

Reply:
<img src="https://media.giphy.com/media/80TEu4wOBdPLG/giphy.gif" />